content/0006-integration-layout/rfc.md

```diff
@@ -283,12 +280,37 @@ summary: Create an issue
 description: |
   Creates a new issue (if issues are enabled).
 
-# URL or path relative to this file to an icon or icons representing this
+# URL or path relative to this file to an icon representing this container
 # image. Optional. Defaults to the integration icon.
 icon: media/ticket.svg
-#icon:
-#  tiny: media/ticket-tiny.svg
-#  medium: media/ticket.svg
+
+# A list of usage examples for this container image. Optional.
+examples:
+-
+  # A descriptive name for this example. Required.
+  summary: Create an issue by accepting user input for the issue title
+
+  # The YAML content of the example. Required.
+  content:
+    # The schema version for the example. Required. Must be the exact string
+    # "v1".
+    apiVersion: v1
+
+    # The schema kind for the example. Required. Must be one of "Workflow" for
+    # an example containing a complete or partial workflow, "Step" for an
+    # example containing only step configuration, or "Trigger" for an example
+    # containing only trigger configuration.
+    kind: Step
```

Comment:
Don't we know the 'kind' implicitly from the location of the example in the repo?

Reply:
I've seen a couple of places where it was better to provide a full example of a workflow, even for an individual step/trigger, which is why I included this. For example, for some triggers it really only makes sense to show how they work in the context of binding to parameters and being used in a step.

Reply:
That's a good point. I think we're not going to be able to avoid schema-generated step/trigger snippets forever. The need would be for a modular unit of code you could append in the right spot in a generated workflow, for example if we built a hypothetical drag-and-drop editor (it's not that hypothetical if you talk to Sig and Kenaz). 

I was originally thinking the examples could serve that purpose temporarily, before we figure out the schema issues we identified, but maybe its unwise to use human-generated content for that purpose anyways.